tests/test_s3/test_s3_utils.py

```diff
@@ -9,6 +8,12 @@
     clean_key_name,
     undo_clean_key_name,
 )
+import sys
+
+if sys.version_info.major == 2 and sys.version_info.minor == 7:
```

Comment:
You can use `if six.PY2` for this:
https://github.com/spulec/moto/blob/c902de8a098d2537792ec5533feb01085febd5ea/moto/s3/utils.py#L97

Reply:
Thanks. I will use that

Reply:
Might want to move this conditional import to `moto.compat` and then import from there.

Reply:
@bpandola The import is for unittests, personaly I would not move that to `moto.compat`. I could create a `tests.compat` if required?

Reply:
Good point @bpandola. Yeah, if you could, @Codeglitches, that would be useful

Reply:
@bblommers do you want it in `moto.compat` or `tests.compat`?

Reply:
Sorry, that wasn't clear - I meant create a `tests.compat`

Reply:
I created `tests.compat.py` and used the same import style as in moto.compat (with a try...except)